specificatie/genereervariant/openapi.yaml

```diff
@@ -302,13 +302,11 @@ paths:
     put:
       tags:
       - Beheren volgindicaties
-      summary: Toevoegen, wijzigen of beëindigen volgindicatie
+      summary: Plaats, wijzig of beëindig een volgindicatie
       description: |
-        Toevoegen, wijzigen of beëindigen van een volgindicatie van een specifieke persoon. Wanneer er nog geen volgindicatie is op deze persoon wordt de volgindicatie toegevoegd. Wanneer er al een volgindicatie is op deze persoon, wordt de volgindicatie gewijzigd.
-
-        De einddatum op een volgindicatie kan worden verwijderd (leeg gemaakt) door een leeg object { } te sturen in de request body.
-
-        Een volgindicatie kan worden beëindigd door het sturen van einddatum gelijk aan de datum van gisteren.
+        Plaats, wijzig of beëindig een volgindicatie op een specifieke persoon. Als je de persoon nog niet volgt, wordt een volgindicatie geplaatst. Als je de persoon al wel volgt, wordt de volgindicatie gewijzigd.
+        Je kunt de einddatum van een volgindicatie verwijderen (leeg maken) door in de request body een leeg object { } te sturen.
```

Comment:
wordt: Verwijder de einddatum van een volgindicatie door in de request body een leeg object { } te sturen.